code/modules/tgui_panel/external.dm

```diff
@@ -1,13 +1,13 @@
 /**
- * Copyright (c) 2020 Aleksej Komarov
- * SPDX-License-Identifier: MIT
- */
+  * Copyright (c) 2020 Aleksej Komarov
+  * SPDX-License-Identifier: MIT
+  */
 
 /client/var/datum/tgui_panel/tgui_panel
 
 /**
- * tgui panel / chat troubleshooting verb
- */
+  * tgui panel / chat troubleshooting verb
+  */
```

Comment:
Are you kidding me...

Proper documentation comment must be formatted like this:

```js
/**
 * text
 */
```

Anything other than this is HERESY and should be purged with fire. I can name you 8 languages where this is a common standard for doc blocks, and there is no reason DM should be a snowflake in this question.

Reply:
According to both the DMdoc readme by Spacemaniac (https://github.com/SpaceManiac/SpacemanDMM/blob/master/src/dmdoc/README.md)
And the TGstation Documentation Guidelines by Oranges (https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)

The correct way to format is 
```
/**
  * # Fubar
  */
/obj/foo
```
And if we check for instances of 
(this is including other code using space indentation)
one space:

![reg1](https://user-images.githubusercontent.com/57223640/97193988-e98a5100-17a9-11eb-90b8-4da6a2e4f280.PNG)

as opposed to two spaces:
![reg2](https://user-images.githubusercontent.com/57223640/97194011-edb66e80-17a9-11eb-87ae-ca29b1bda31c.PNG)

2 spaces used more than twice as often used as it stands.

Fell free to make a PR changing Guidelines to one space and I'll change the PR but until then this is the correct way to do it

Reply:
@TiviPlus #54623